pycloudlib/azure/cloud.py

```diff
@@ -685,13 +716,30 @@ def launch(
                 "Found network interface: %s. Reusing it", nic.name
             )
 
+        vm_params = kwargs.get("vm_params", {})
+        sec_type = security_type.value()
+        sec_type.configure_vm_params(
+            vm_params=vm_params,
+            **kwargs.get("security_type_params"),
```

Comment:
If `security_type_params` is not in kwargs, then we would try to expand a `**None` which is invalid python.

```suggestion
            **kwargs.get("security_type_params", {}),
```

Probably worth unit-testing this? What do you think?